my-app/src/App.css

```diff
@@ -53,5 +53,44 @@
 }
 
 .navbar{
-  box-shadow:10px;
+  background-color: black;
+  color: #fff;
+  font-weight: bold;
+}
+
+.logo{
+  display: flex;
+  color: #61DAFB;
+}
+
+.maincontent{
+  background-color: rgba(29, 27, 27, 0.8);
+  padding: 10px; 
+  color: #fff;
+  font-weight: bold;
+  background-image: url(C:\Users\chidima\OneDrive\Documents\React-JS\my-app\public\logo192.png );
```

Comment:
you know if you deploy this to  a server this will fail because it's pointing to your system so you need to put the url in such a way that it takes the context of where ever the app is running and add the path to it. don't use the full system path that's wrong

Reply:
ok